tests/pipelines/test_pipelines_common.py

```diff
@@ -576,15 +576,23 @@ def test_from_pipe_consistent_config(self):
 
     def test_from_pipe_consistent_forward_pass(self, expected_max_diff=1e-3):
         components = self.get_dummy_components()
-        original_expected_modules, _ = self.original_pipeline_class._get_signature_keys(self.original_pipeline_class)
 
+        pipe = self.pipeline_class(**components)
```

Comment:
updated this test so that we will be able to catch this kind of error ((https://github.com/huggingface/diffusers/pull/7564/files#r1548196510) in fast test (did not know that the offloading test is slow test)